src/stories/containers/Finances/utils/types.ts

```diff
@@ -41,3 +41,8 @@ export interface MomentDataItem extends Team {
   totalActuals: number;
   lastModified: DateTime;
 }
+export type Metrics = 'Budget' | 'Actual' | 'Forecast' | 'Net Expenses On-chain' | 'Net Expenses Off-chain';
+export type MetricsWithAmount = {
```

Comment:
use `interface` for consistency as it is an object

Reply:
you are right, done